omero/sysadmins/server-ldap.txt

```diff
@@ -211,21 +227,28 @@ that is, if your URL is of the form ``ldaps://ldap.example.com:636`` you may
 need to configure a key and trust store for Java. See the
 :doc:`server-security` page for more information.
 
+.. _synchronizing-ldap:
+
 Synchronizing LDAP on user login
 --------------------------------
 
 This feature allows for LDAP to be considered the authority on user/group
-membership. With the following settings enabled each time a user logs in to
+membership. With the following setting enabled, each time a user logs in to
 OMERO their LDAP groups will be read from the LDAP server and reflected in
-OMERO. Enabling this will result in any bespoke OMERO groups that have been
-created being removed from the user's profile. The groups will still exist on
-the server but the association between user and group will not be reflected
-unless such a link is made in LDAP.
-
-::
+OMERO::
 
     bin/omero config set omero.ldap.sync_on_login true
 
+Admin actions carried out in the clients may not survive this
+synchronization e.g. if an admin has removed an LDAP user from an LDAP group
+in the UI, the user will be re-added to the group when logging in again after
+the synchronization.
+
+.. note:: This applies to groups created by LDAP in OMERO 5.1.x. Groups
+    created in older versions of OMERO will not be registered as LDAP groups
+    if you have manually altered their membership, even if the membership now
+    matches the LDAP group.
+
```

Comment:
`bin/omero ldap setdn true --group-name $NAME` can be used to make the previous ("OMERO?") group an LDAP group.